src/mesh/Mesh.cpp

```diff
@@ -170,14 +170,23 @@ const PtrData &Mesh::data(DataID dataID) const
   return *iter;
 }
 
-bool Mesh::hasDataName(const std::string &dataName) const
+bool Mesh::hasDataName(std::string_view dataName) const
```

Comment:
I assume that the change to `std::string_view` is actually related to this PR?

Reply:
This allows to pass the string_view straight through. Otherwise we have to create a std::string.